libs/designsystem/src/lib/components/segmented-control/segmented-control.component.stories.ts

```diff
@@ -1,5 +1,5 @@
 import type { Meta, StoryObj } from '@storybook/angular';
-
+/* eslint-disable-next-line no-restricted-imports */
```

Comment:
How come the import (on the next line) is different from the other import statements in (most/all) the other stories?

Reply:
Segmented Control is the only component that has not yet been converted to a secondary entry point, so for now it is just referenced with a relative path and compiled by storybook.